.happy/terraform/modules/ecs-stack/variables.tf

```diff
@@ -35,6 +35,18 @@ variable happy_config_secret {
   description = "Happy Path configuration secret name"
 }
 
+variable "wmg_disk_cache_path" {
+  type        = string 
+  description = "Absolute path of directory on the local disk where WMG data snapshot is cached"
+  default     = "/tmp/wmg_disk_cache"
```

Comment:
Doesn't seem necessary - no one is calling this right now.

Reply:
@ebezzi - I removed this